### ISSUE TITLE
IllegalStateException in MediaController.notifyAccepted()

### ISSUE THREAD

**Issue by h6ah4i** [androidx/media#49]
Hi. I noticed the following crash report in Crashlytics for my app.

## Example stack trace (copied from Firebase Crashlytics)

```
Fatal Exception: java.lang.IllegalStateException
       at androidx.media3.common.util.Assertions.checkState(Assertions.java:85)
       at androidx.media3.session.MediaController.notifyAccepted(MediaController.java:1800)
       at androidx.media3.session.MediaControllerImplLegacy.updateControllerInfo(MediaControllerImplLegacy.java:1357)
       at androidx.media3.session.MediaControllerImplLegacy.handleNewLegacyParameters(MediaControllerImplLegacy.java:1318)
       at androidx.media3.session.MediaControllerImplLegacy.access$500(MediaControllerImplLegacy.java:109)
       at androidx.media3.session.MediaControllerImplLegacy$ControllerCompatCallback.onSessionReady(MediaControllerImplLegacy.java:1559)
       at android.support.v4.media.session.MediaControllerCompat$Callback$MessageHandler.handleMessage(MediaControllerCompat.java:1121)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:214)
       at android.app.ActivityThread.main(ActivityThread.java:7386)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:980)
```

## Where the exception thrown

https://github.com/androidx/media/blob/f92ae23dd83533bce93c7a937f2fa40ce810f207/libraries/session/src/main/java/androidx/media3/session/MediaController.java#L1800

## Library versions
- `androidx.media3`: Built from source (Commit ID: f92ae23dd83533bce93c7a937f2fa40ce810f207)
- `androidx.media`: v1.5.0

## Affected Android versions  (obtained info from Firebase Crashlytics)

- Android 8, 9, 10, 11 and 12 
(older OS might also be affected)

## How to reproduce the issue?

Sorry, I have not succeeded in reproducing the issue yet. I'll post details if I can get any info.

**Comment by entibaldi** [comment]
We are also reproducing this issue for several of our users

**Comment by h6ah4i** [comment]
@tonihei Thanks for the fix in 1.0.0-beta03. I no longer see the app crashes due to this error anymore :+1:
